.github/workflows/cloud-tpu-ci-nightly.yml

```diff
@@ -16,8 +16,8 @@ jobs:
     strategy:
       fail-fast: false  # don't cancel all jobs on failure
       matrix:
-        python-version: ["3.10"]  # TODO(jakevdp): update to 3.11 when available.
-        jaxlib-version: [latest, nightly]
+        python-version: ["3.10"] # TODO(jakevdp): update to 3.11 when available.
+        jaxlib-version: ["latest-release", "nightly"]
```

Comment:
Can we add more info about what latest-release actually means? 

Maybe:

`jax (head), jaxlib/libtpu (pypi)`

Reply:
It seems a little weird to put an unused variable in the matrix... can we just assume that unless otherwise noticed, all jax CI is running jax at HEAD? It's a bit odd to run anything else :)

Reply:
Okay, but what I said is True? Latest release does mean that?

Reply:
Ooh I missed the part about changing it to pypi. I actually think latest-release is more accurate. Pypi is just where we host releases, including non-latest releases.

Reply:
But yes, jaxlib-version==latest-release does still mean we run jax at head. Just jaxlib is from a release.